modules/ROOT/pages/_partials/secrets-manager.adoc

```diff
@@ -232,12 +232,12 @@ Besides the default flag `--help`, this command also accepts:
 [%header,cols="30a,40a,30a"]
 |===
 |Flag | Description |  Example
-|`--group-id`| Mandatory. Secret group ID. | `--group-id 030493`
+|`--group-id`| Mandatory. Secret group ID. | `--group-id 1fec0a49-1551-4199-bfcc-cf0352d0f29d`
 |`--name`|Mandatory. Name for your secret. | `--name TestSecret`
 |`--type` | Mandatory. Choose the certificate secret type. +
 Options: PEM. | `--type PEM`
-|`--cert-file` |Certificate file type | `--cert-file json`
-|`--expiration-date`| Expiration date for the secret. | `--expiration-date 01/01/2025`
+|`--cert-file` |Certificate file path | `--cert-file /example-cert.pem`
```

Comment:
Minor - The example file path must be  `./example-cert.pem`  (period must be present before the slash)